src/libs/SidebarUtils.ts

```diff
@@ -117,12 +117,22 @@ function getOrderedReportIDs(
     betas: Beta[],
     policies: Record<string, Policy>,
     priorityMode: ValueOf<typeof CONST.PRIORITY_MODE>,
-    allReportActions: OnyxCollection<ReportAction[]>,
+    allReportActions: OnyxCollection<ReportActions>,
+    transactionViolations: TransactionViolations,
 ): string[] {
+    const countActions = (actions: ReportActions | null): number => (actions ? Object.keys(actions).length : 0);
+
```

Comment:
This is still overly complicated to take the length of an array
```suggestion
```

Reply:
We can get this down to `Object.keys(allReportActions[reportIDKey] ?? {}).length`

Reply:
I think that's what I suggested below